terraform/eks/main.tf

```diff
@@ -81,47 +57,46 @@ provider "kubernetes" {
   load_config_file       = false
 }
 
-# create a unique namespace for each run
-resource "kubernetes_namespace" "aoc_ns" {
-  metadata {
-    name = "aoc-ns-${module.common.testing_id}"
-  }
-}
-resource "kubernetes_config_map" "aoc_config_map" {
-  metadata {
-    name      = "otel-config"
-    namespace = kubernetes_namespace.aoc_ns.metadata[0].name
-  }
-
-  data = {
-    "aoc-config.yml" = module.basic_components.otconfig_content
-  }
+provider "kubectl" {
```

Comment:
would every testcase need kubectl? 
but i think it's okay to leave it created since it makes not hurts and save "ifelse".

Reply:
No. Currently only container insight infra tests would use `kubectl` provider. As providers are global-wise configuration, I would prefer to have them stayed in `main.tf` to avoid duplicated declaration in the future. That's my original thinking.